pkgs/development/python-modules/pep257/default.nix

```diff
@@ -1,22 +1,50 @@
-{ lib, buildPythonPackage, fetchFromGitHub, pytestCheckHook, mock }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mock
+, pytestCheckHook
+, pythonOlder
+, snowballstemmer
+, toml
+}:
 
 buildPythonPackage rec {
   pname = "pep257";
   version = "6.1.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "GreenSteam";
     repo = "pep257";
     rev = version;
-    sha256 = "0hcf3nyvzl8kd6gmc9qsiigz7vpwrjxcd1bd50dd63cad87qqicg";
+    hash = "sha256-j0WMD2qKDdMaKG2FxrrM/O7zX4waJ1afaRPRv70djkE=";
   };
 
-  checkInputs = [ pytestCheckHook mock ];
+  propagatedBuildInputs = [
+    snowballstemmer
+    toml
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    mock
+  ];
+
+  disabledTestPaths = [
+    # Tests run pip install
+    "src/tests/test_integration.py"
+  ];
+
+  pythonImportsCheck = [
+    "pydocstyle"
+  ];
 
   meta = with lib; {
-    homepage = "https://github.com/GreenSteam/pep257/";
     description = "Python docstring style checker";
-    longDescription = "Static analysis tool for checking compliance with Python PEP 257.";
+    longDescription = "Static analysis tool for checking compliance with Python PEP 257";
+    homepage = "https://github.com/GreenSteam/pep257/";
```

Comment:
This redirects to https://github.com/PyCQA/pydocstyle. Are the packages merged now? Can we drop this?

Reply:
I think that `pep257` can be dropped. `pytest-pep257` would need it but that package was not maintained for 6 years.